sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareErrorCode.java

```diff
@@ -155,7 +155,8 @@
     /**
      * Static value MultipleConditionHeadersNotSupported for ShareErrorCode.
      */
-    public static final ShareErrorCode MULTIPLE_CONDITION_HEADERS_NOT_SUPPORTED = fromString("MultipleConditionHeadersNotSupported");
+    public static final ShareErrorCode MULTIPLE_CONDITION_HEADERS_NOT_SUPPORTED
+        = fromString("MultipleConditionHeadersNotSupported");
```

Comment:
Was this change due to Checkstyle issues? Given that this is codegen code I would just have the check ignore this file.

Reply:
I'll update the base branch to just ignore so I dont need to do it in every PR